src/exchange/information/information.html

```diff
@@ -4,257 +4,205 @@
-                                                     data-align="end"
-                                                     data-disabled="ctrl.exchangeServiceInfrastructure.is25g()"
-                                                     data-ng-if="ctrl.displayOrderDiskSpace()">
-                                        <oui-action-menu-item data-on-click="ctrl.orderDiskSpace()">
-                                            <span data-translate="exchange_action_order_space_disk"></span>     
-                                        </oui-action-menu-item>
-                                    </oui-action-menu>
-                                </li>
-                            </ul>
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'exchange_tab_INFORMATIONS_stats' | translate}}">
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_domains' | translate}}">
+                            <oui-tile-description>
+                                <span data-ng-bind="ctrl.exchange.domainsNumber"></span>
+                            </oui-tile-description>
+                        </oui-tile-definition>
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_accounts' | translate}}">
+                            <oui-tile-description>
+                                <span data-ng-bind="ctrl.exchange.accountsNumber"></span>
+                            </oui-tile-description>
+                        </oui-tile-definition>
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_tasks' | translate}}">
+                            <oui-tile-description>
+                                <span data-ng-bind="ctrl.exchange.tasksNumber"></span>
+                            </oui-tile-description>
+                        </oui-tile-definition>
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_disk_usage' | translate}}"
+                                             data-ng-if="ctrl.exchange.diskPercent"
+                                             data-term-popover="{{:: 'exchange_action_order_space_disk_tooltip' | translate}}">
+                            <oui-tile-description>
+                                <oui-progress data-min-value="0"
+                                              data-max-value="100">
+                                    <oui-progress-bar data-ng-if="ctrl.exchange.diskPercent < 51"
+                                                      data-type="success"
+                                                      data-value="ctrl.exchange.diskPercent">
+                                        <span data-ng-bind="ctrl.exchange.currentDiskUsage.value + ' ' + ('unit_size_'+ ctrl.exchange.currentDiskUsage.unit | translate) + ' / ' + ctrl.exchange.totalDiskSize.value + ' ' + ('unit_size_'+ ctrl.exchange.totalDiskSize.unit | translate)"></span>
```

Comment:
Maybe this would be easier to read if you had : 
```suggestion
                                        <span data-ng-bind="getFormattedDiskUsage()"></span>
```
And in controller : 
```js
 getFormattedDiskUsage () {
     return `${this.exchange.currentDiskUsage.value} ${this.$translate.instant(this.exchange.currentDiskUsage.unit)} / ...`
 }
```

Reply:
done 😄